src/Core/src/Platform/iOS/NativePicker.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using Foundation;
+using UIKit;
+using ObjCRuntime;
+using RectangleF = CoreGraphics.CGRect;
+
+namespace Microsoft.Maui
+{
+	public class NativePicker : NoCaretField
+	{
+		readonly HashSet<string> _enableActions;
+
+		public NativePicker(UIPickerView? uIPickerView)
+		{
+			UIPickerView = uIPickerView;
+
+			string[] actions = { "copy:", "select:", "selectAll:" };
+			_enableActions = new HashSet<string>(actions);
+		}
+
+		public UIPickerView? UIPickerView { get; set; }
+
+		public override bool CanPerform(Selector action, NSObject? withSender)
+			=> _enableActions.Contains(action.Name);
+	}
+
+	public class NoCaretField : UITextField
```

Comment:
Maybe this should be in a separate file since I saw this in the time pickers PR?

Reply:
Created a separate file.